src/ameba/rule/lint/require_parentheses.cr

```diff
@@ -0,0 +1,50 @@
+module Ameba::Rule::Lint
+  # A rule that disallows method calls with at least one argument, where no
+  # parentheses are used around the argument list, and a logical operator
+  # (`&&` or `||`) is used within the argument list.
+  #
+  # For example, this is considered invalid:
+  #
+  # ```
+  # if foo.includes? "bar" || foo.includes? "batz"
+  # end
+  # ```
+  #
+  # And need to be written as:
+  #
+  # ```
+  # if foo.includes?("bar") || foo.includes?("batz")
+  # end
+  # ```
+  #
+  # YAML configuration example:
+  #
+  # ```
+  # Lint/RequireParentheses:
+  #   Enabled: true
+  # ```
+  class RequireParentheses < Base
```

Comment:
would be nice to have a corrector for this one.

Reply:
I agree, although it might be added in followup PR. Also, I believe it's better to fix the bugs that popped up recently before adding even more rules that might trip on 'em.

Reply:
What bugs are you referring to? I'm fine either adding it now or later on, though may take some time to figure out the best way to do it.

Reply:
@nobodywasishere #489 to be exact. I'd prefer adding this l8r on tbh.

Reply:
Either way works. Don't expect 1.7.0 to come out for a while either. Just don't want this (and other PRs) to bit rot.